types/ChallengeBuilder.tsx

```diff
@@ -40,9 +40,9 @@ export const buildChallenge = async (
 export const createChallenge = async (
   firestoreCtrl: FirestoreCtrl,
   challenge_name: string,
-  date: Date,
   description: string,
   location: LocationObject | null,
+  date?: Timestamp,
```

Comment:
I think it would be a better practice to store all fields into Firebase, even if they would be "null", instead of letting them be optional

Reply:
Testing says otherwise!